main.py

```diff
@@ -470,37 +533,17 @@ def main(cl_arguments):
 
     if args.do_target_task_training:
         # Train on target tasks
-        task_names_to_avoid_loading = setup_target_task_training(args, target_tasks, model, strict)
-        if args.transfer_paradigm == "frozen":
-            # might be empty if elmo = 0. scalar_mix_0 should always be
-            # pretrain scalars
-            elmo_scalars = [
-                (n, p)
-                for n, p in model.named_parameters()
-                if "scalar_mix" in n and "scalar_mix_0" not in n
-            ]
-            # Fails when sep_embs_for_skip is 0 and elmo_scalars has nonzero
-            # length.
-            assert_for_log(
-                not elmo_scalars or args.sep_embs_for_skip,
-                "Error: ELMo scalars loaded and will be updated in do_target_task_training but "
-                "they should not be updated! Check sep_embs_for_skip flag or make an issue.",
-            )
-        for task in target_tasks:
+        pre_target_train_path = setup_target_task_training(args, target_tasks, model, strict)
+
+        for index in range(len(target_tasks)):
```

Comment:
for index, task in enumerate(target_tasks):

This lets you skip the extra indexing line.